data/campaigns/Under_the_Burning_Suns/scenarios/11_Battle_for_Zocthanol_Isle.cfg

```diff
@@ -180,8 +180,8 @@
 
         [ai]
             recruitment_pattern=scout, fighter, archer, fighter, mixed fighter
-            aggression=0.8
-            caution=0.1
+            aggression={ON_DIFFICULTY 0.6 0.7 0.8}
+            caution={ON_DIFFICULTY 0.3 0.2 0.1}
```

Comment:
Many places would benefit from comments about what the intended effect is, what play styles is it meant to be more or less lenient for.

Reply:
basically here when that side was more aggressive, it was killing me more; the idea is to have them back off a bit on easier difficulties to give the player a bit more breathing room

Reply:
comment added in [dd0c083](https://github.com/wesnoth/wesnoth/pull/8421/commits/dd0c08340433d9d125b45e46e014426d46b0fdc3).